style/layers.js

```diff
@@ -41,6 +41,12 @@ americanaLayers.push(
 
   layerHighwayShieldInterstate,
 
+  layerBoundaryCountyBg,
+  layerBoundaryCounty,
+  layerBoundaryStateBg,
+  layerBoundaryState,
+  layerBoundaryCountryBg,
+  layerBoundaryCountry,
   layerPlaceCity,
```

Comment:
I think we should put the boundary layer between water and roads.  One of the screen shots has a boundary going through a highway shield which doesn't look right.  Note that the layers are applied in the order listed.

Reply:
Love this suggestion. The roads pop out more when the border does not go over them.